src/main/java/io/github/thebusybiscuit/slimefun4/utils/WorldUtils.java

```diff
@@ -0,0 +1,29 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+
+import javax.annotation.Nonnull;
+
+import io.github.thebusybiscuit.slimefun4.api.MinecraftVersion;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import org.bukkit.World;
+
+/**
```

Comment:
Seeing as you did this in other files
```suggestion
/*
```

Reply:
Those are javadocs. 2 asteriks is correct...
The other instances where multiline-comments.

Reply:
`/**` indicates a JavaDoc comment. Sometimes in the past Cookie has put this type of comment instead of a multi-line comment in the code itself (rather than on top of methods or classes). This is why I remove 1 star. Here is is valid

Reply:
o my bad